### PR TITLE
initial PR

### PR DESCRIPTION
**Pull request by gattytto**
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,9 @@
...
-
-This repository uses an automatic GitHub Action which will build, tag, and push a container image on every push. In order to specify which version of the container is built, tagged, and pushed, be sure to edit the VERSION file with the correct version.
-
- - To update the Dockerfile for the latest version, make a PR against master
- - Ensure VERSION contains the correct version you wish to build
- - To update an older version of the sidecar container, open a PR against the branch for the version you wish to update
-
- # Platforms
- This repository supports multiple architecture builds. To add a platform, edit the PLATFORMS file and add your desired platform. Currently this plugin sidecar supports the following architectures:
- - linux/amd64
```
Comment (ericwill) [inline README.md:20]: This was here for a reason. :) we are not using the branch system any longer, the latest version lives on the master branch, after that old versions get pushed to their own branches. You can take a look at the [openshift sidecar repo](https://github.com/che-dockerfiles/che-sidecar-openshift-connector) for an example.

File: etc/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```
Comment (ericwill) [inline etc/entrypoint.sh:1]: Make sure `chmod +x` this file otherwise we will have failures.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2019 Red Hat, Inc.
```
Comment (ericwill) [inline Dockerfile:1]: Copyright year

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM quay.io/buildah/stable:v1.14.3
```
Comment (ericwill) [inline Dockerfile:11]: Is there enough overlap to justify using the `che-sidecar-kubernetes-tooling` image as a base?

**Review by ericwill — CHANGES_REQUESTED** [review]
Please see my comments on the README about the repo layout. We also need the following files:
* `VERSION`
* `PLATFORMS`

You can look at [this repo](https://github.com/che-dockerfiles/che-sidecar-openshift-connector) for inspiration.

**Comment by gattytto** [comment]
@ericwill ok I'll make a new pr based on the changes you mentioned

**Pull request closed without merging**